src/rabbit_channel.erl

```diff
@@ -425,14 +434,15 @@ handle_call(_Request, _From, State) ->
 
 handle_cast({method, Method, Content, Flow},
             State = #ch{reader_pid   = Reader,
-                        virtual_host = VHost}) ->
+                        interceptor_state = IState}) ->
     case Flow of
         flow   -> credit_flow:ack(Reader);
         noflow -> ok
     end,
-    try handle_method(rabbit_channel_interceptor:intercept_method(
-                        expand_shortcuts(Method, State), VHost),
-                      Content, State) of
+    Method1 = expand_shortcuts(Method, State),
```

Comment:
`expand_shortcuts` might throw an error, so this here makes the channel crash when no queue name is specified for things like `basic.consume` or `queue.bind` https://github.com/rabbitmq/rabbitmq-server/blob/master/src/rabbit_channel.erl#L745